site/content/docs/11-deploying.md

```diff
@@ -5,18 +5,18 @@ title: Deployment
 Sapper apps run anywhere that supports Node 8 or higher.
 
 
-### Deploying to Now
+### Deploying to Vercel (previously Now)
 
-> This section relates to Now 1, not Now 2
+> This section relates to Vercel 1, not Vercel 2
```

Comment:
Does this section really related to Vercel 2? I'm not that familiar with their platform, but I'd be surprised if their command line tool uses the old version by default. I'd expect them to be promoting the latest